students/Hebdowska_Grazyna/lesson_06/countries_and_cites.py

```diff
@@ -0,0 +1,31 @@
+def input_files():
```

Comment:
Function name 'input_files' is misleading

Consider to use *args for undefined number of arguments"
```
number_of_countries = int(input('How many countries: '))
for i in range(number_of_countries ):
    country, *cities = input("Country and cities: ").split(" ")
    countries[country] = cities
```